js/package.json

```diff
@@ -6,9 +6,9 @@
   "dependencies": {
     "@flarum/prettier-config": "^1.0.0",
     "flarum-tsconfig": "^1.0.2",
-    "flarum-webpack-config": "^1.0.0",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^4.9.1"
+    "flarum-webpack-config": "^2.0.0",
```

Comment:
We'll need to bump core's constraint to 1.2+

Reply:
Already present in `master` :)

Reply:
Oop, my bad